client/lib/sites-list/list.js

```diff
@@ -85,7 +85,7 @@ SitesList.prototype.fetch = function() {
 			fields:
 				'ID,URL,name,capabilities,jetpack,visible,is_private,is_vip,icon,plan,jetpack_modules,single_user_site,is_multisite,options', //eslint-disable-line max-len
 			options:
-				'is_mapped_domain,unmapped_url,admin_url,is_redirect,is_automated_transfer,allowed_file_types,show_on_front,main_network_site,jetpack_version,software_version,default_post_format,created_at,frame_nonce,publicize_permanently_disabled,page_on_front,page_for_posts,advanced_seo_front_page_description,advanced_seo_title_formats,verification_services_codes,podcasting_archive,is_domain_only,default_sharing_status,default_likes_enabled,wordads,upgraded_filetypes_enabled,videopress_enabled,permalink_structure,gmt_offset', //eslint-disable-line max-len
+				'is_mapped_domain,unmapped_url,admin_url,is_redirect,is_automated_transfer,allowed_file_types,show_on_front,main_network_site,jetpack_version,software_version,default_post_format,created_at,frame_nonce,publicize_permanently_disabled,page_on_front,page_for_posts,advanced_seo_front_page_description,advanced_seo_title_formats,verification_services_codes,podcasting_archive,is_domain_only,default_sharing_status,default_likes_enabled,wordads,upgraded_filetypes_enabled,videopress_enabled,permalink_structure,gmt_offset,signup_is_store', //eslint-disable-line max-len
```

Comment:
It would be nice to split these monster lists into a joined array:
```js
OPTIONS = [
  'is_mapped_domain',
  'unmapped_url',
].join( ',' );
```
If nothing else, it lets the reviewer to see what was actually added to the list 🙂

Reply:
Heh, I was also surprised about that. But I'm not touching that in this PR. :D